packages/main/src/types/SegmentedButtonMode.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Different SegmentedButton modes.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.SegmentedButtonMode
+ */
+enum SegmentedButtonMode {
+	/**
+	 * There is always one selected. Selecting one deselects the previous one.
+	 * @public
+	 * @type {Single}
+	 */
+	Single = "Single",
```

Comment:
After discussing this internally, we would liek to be consistent with List and Table mode property and enum values

I suggest renaming the values to:
Single - > SingleSelect
Multi -> MultiSelect

Reply:
done